src/controllers/auth.js

```
@@ -81,6 +81,19 @@ const refreshAccessToken = async (req, res) => {
   res.status(200).json(tokens)
 }
 
+const confirmEmail = async (req, res) => {
+  const confirmToken = req.params.confirmToken
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)